docs/self-managed/platform-deployment/helm-kubernetes/upgrade.md

```diff
@@ -89,6 +89,21 @@ For more details on the Keycloak upgrade path, you can also read the [Bitnami Ke
 
 ## Version update instructions
 
+### v8.2.3
+
+:::caution Breaking Change
+
+Zeebe Gateway authentication is now enabled by default. To authenticate,
```

Comment:
I'd suggest to add just 1 line in the the caution section which is:
> Zeebe Gateway authentication is now enabled by default.

then out of it adding all details.

otherwise, if we have a breaking change with more details it will look odd.